src/main/kotlin/mods/eln/sixnode/EmergencyLamp.kt

```diff
@@ -78,11 +78,11 @@ class EmergencyLampDescriptor(name: String, val cable: ElectricalCableDescriptor
     override fun addInformation(itemStack: ItemStack?, entityPlayer: EntityPlayer?, list: MutableList<String>,
                                 par4: Boolean) {
         with(list) {
-            add("As long as power is provided, the internal battery")
-            add("is charged and the lamp is off. On a power failure,")
-            add("the lamp turns on and runs on batteries.")
-            add(Utils.plotVolt("Nominal voltage:", cable.electricalNominalVoltage))
-            add(Utils.plotEnergy("Battery capacity:", batteryCapacity))
+            add(tr("As long as power is provided, the internal battery"))
+            add(tr("is charged and the lamp is off. On a power failure,"))
+            add(tr("the lamp turns on and runs on batteries."))
+            add(Utils.plotVolt(tr("Nominal voltage:"), cable.electricalNominalVoltage))
+            add(Utils.plotEnergy(tr("Battery capacity:"), batteryCapacity))
```

Comment:
Considering this is here like this, I bet we can use it in the other places we have it the other way above, for consistency. (I think it makes more sense to have the translation before the Utils call but perhaps that's not a great idea)